simplified Chinese translations and minor

VeriTas-arch opened the pull request:
- Complete simplified Chinses translations (relative to en_us).
- Add some useful tools for translators in the `.github/tools` folder.
  - These two scripts are mianly for checking which keys aren't translated and sorting the json keys so as to improve readability. In previous issues I think translators are frequently faced with these two tasks, and the tools can perhaps give them a hand. 
- Minor prettifying works on some markdown files.
  - Really minor changes. Simply to make them prettier in text editors.

VeriTas-arch committed:
simplified Chinese translations and minor

- Complete simplified Chinses translations (relative to en_us)
- Add some useful tools for translators
- Minor prettifying works on some markdown files.

Noaaan commented on CREDITS.md:
Compared to other markdown files this was updated to use `*` for unordered lists instead of `-`. Is there any specific reason the file was changed this way?
The only reason it uses these to begin with is historical - Nowadays I personally just use dashes, despite markdown allowing for stars

VeriTas-arch replied:
I suppose it results from the auto format setting of markdown plugin. `*` are used simply because they are the majority. Thanks for the review, they are recovered to `-` now.

Noaaan commented on PATCHNOTES.md:
I am not sure whether it is a good idea to go forth changing this file this extensively. I assume this was simply formatted with a linter, as in its current state it breaks the display, as seen in GitHubs markdown renderer:

![image](https://github.com/user-attachments/assets/8c1e1cbf-34e7-4d40-884e-d37e5da0d349)

Making all these changes also dillutes the history/blame of the file, which I am planning to use for the Mythic Metals Wiki later (looong term I plan on having a history section of sorts). Having to manually scrape this data/dates from commit history later is less ideal than just being able to use the changes + dates in the patch notes today.

Noaaan replied:
Sidenote - did you know a double space on the end of a line counts as a line break? This is why these old patch ntoes break.

VeriTas-arch replied:
O, that's quite an oversight. Generally I would favor unordered lists starting with `-` to be main parts of changelog, since most changes are quite *scattered*. Double space is never used since I prefer no trailling space, although it is indeed legal in usage. The file is now recovered unchanged compared to the last commit. Well, hopefully it can be fixed with a better format to improve readability, which is a similar problem in the wiki (I think that it is better in readability to leave an empty line rather than double sapce so as to get a line break. If to be more `html-style`, then `<br>` is also an alternative).

Noaaan replied:
Current changelogs should not have this problem as I now use proper lists and headers over abusing double space. Especially since the formatting is pushed directly to CurseForge, GH releases, and Modrinth. Thank you for understanding + reverting

Formatting will obviously not be 1:1 when this information is added to the wiki, it will look nicer 🙂

VeriTas-arch committed:
changes requested by review.

- replace `*` in CREDITS.md with `-`
- recover PATCHNOTES.md to be unchanged

VeriTas-arch committed:
minor fix

VeriTas-arch committed:
fix README

Recover the format while trying to achieve a balance between better readability and consistency with former format.

VeriTas-arch commented on README.md:
I think it is similar in rendering while beter in maintaining now?

Noaaan replied:
lgtm

Noaaan commented:
Translation seems fully complete. Nicely done :+1:

The only things missing from it are a few langs from other mods (owo-lib and Additional Entity Attributes), but that is no concern for this PR. I did notice the new 1.21 arrow effects (wind_charged, oozing, infested, and weaving) are missing, but this is native to Mythic Metals itself. I will add the keys for those in main after this PR.

I have attached the respective vanilla translations for these as well: 

![tipped_arrow](https://github.com/user-attachments/assets/b22f5155-a366-4ddd-8846-403c9636c50c)
![tipped_arrow_1_](https://github.com/user-attachments/assets/90ace5f1-1af2-40ef-8093-97123a32c9d9)
![tipped_arrow_2_](https://github.com/user-attachments/assets/251dc998-2b4d-4346-b413-214c2d2976d6)
![tipped_arrow_3_](https://github.com/user-attachments/assets/e604e6bf-fcc1-4136-ad8b-2a27c7c07707)

VeriTas-arch commented:
Tks. Those missing keys can be completed in the next PR.
